lib/trento_web/openapi/schema/host.ex

```diff
@@ -0,0 +1,97 @@
+defmodule TrentoWeb.OpenApi.Schema.Host do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  defmodule IPv4 do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "IPv4",
+      type: :string,
+      format: :ipv4
+    })
+  end
+
+  defmodule IPv6 do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "IPv6",
+      type: :string,
+      format: :ipv6
+    })
+  end
+
+  defmodule Item do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "Host",
+      description: "A discovered host on the target infrastructure",
+      type: :object,
+      properties: %{
+        id: %Schema{type: :integer, description: "Host ID"},
+        # pattern: ~r/[a-zA-Z][a-zA-Z0-9_]+/
```

Comment:
Leftover comment?

Reply:
yes, also a reminder on how to use it :stuck_out_tongue: 
Removing.